gemrb/plugins/CREImporter/CREImporter.cpp

```diff
@@ -710,6 +693,8 @@ void CREImporter::ReadChrHeader(Actor *act)
 		//fallthrough
 	case CREVersion::GemRB:
 		for (int i = 0; i < QSPCount; i++) {
+			// FIXME: why is the data larger than required?
```

Comment:
It just is, we can't do anything about it except change our underlying type and waste a little bit of memory.

Reply:
It has nothing to do with the `act->PCStats->QSlots[i+3]` line? Makes it feel like we should be reading 4 separate bytes into these holes.

Reply:
If it wasn't for the fallthrough, this would only apply to our own unused format and we could just change it. But iwd2 runs this code as well.